config/settings/qa.yml

```diff
@@ -6,3 +6,7 @@ bg_jobs:
     cron: "0 */6 * * *"
     class: "BulkSyncCoursesToFindJob"
     queue: find_sync
+azure:
+  webapp: dummy-webapp
+  rgroup: dummy-rgroup
+  subscription: dummy-subscription
```

Comment:
If just using `config/settings.local.yml` works, let's put these dummy settings in `config/settligs.yml`?